lib/matplotlib/backend_bases.py

```diff
@@ -1109,8 +1109,18 @@ def get_hatch_color(self):
         """
         Gets the color to use for hatching.
         """
+        if self._hatch_color is None:
+            return colors.to_rgba(rcParams['hatch.color'])
         return self._hatch_color
 
+    def set_hatch_color(self, hatch_color):
+        """
+        Gets the color to use for hatching.
+        """
+        if hatch_color is None:
+            raise ValueError()
```

Comment:
Also, don't you want to check that the `hatch_color` provided is a color? That should fix the value error message.